v1/api/index.ts

```diff
@@ -0,0 +1,69 @@
+import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox'
+import fastify, { FastifyBaseLogger, FastifyInstance, RawReplyDefaultExpression, RawRequestDefaultExpression, RawServerDefault } from 'fastify'
+import { siteRoutes } from './sites'
+import multipart from '@fastify/multipart'
+import swagger from '@fastify/swagger'
+import swagger_ui from '@fastify/swagger-ui'
+import { adminRoutes } from './admin'
+import { publisherRoutes } from './publisher'
+
+export type FastifyTypebox = FastifyInstance<
+RawServerDefault,
+RawRequestDefaultExpression<RawServerDefault>,
+RawReplyDefaultExpression<RawServerDefault>,
+FastifyBaseLogger,
+TypeBoxTypeProvider
+>
+
+interface APIConfig {
+  useLogging: boolean
+  useSwagger: boolean
+}
+
+async function apiBuilder ({ useLogging, useSwagger }: Partial<APIConfig>): Promise<FastifyTypebox> {
+  const server = fastify({ logger: useLogging }).withTypeProvider<TypeBoxTypeProvider>()
+  await server.register(multipart) // TODO: discuss whether we want to set a filesize limit
```

Comment:
Is it possible to only register the multipart handler on the publisher post/put endpoints? I don't think we need it running for anything else and it could lead to issues with tmp folders getting spammed.

Reply:
Perfect 👍